lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -1,12 +1,39 @@
 import fnmatch
 import ufe
+import maya.mel as mel
+import maya.cmds as cmds
 import mayaUsd.ufe
 import maya.internal.common.ufe_ae.template as ufeAeTemplate
 
 # We manually import all the classes which have a 'GetSchemaAttributeNames'
 # method so we have access to it and the 'pythonClass' method.
 from pxr import Usd, UsdGeom, UsdLux, UsdRender, UsdRi, UsdShade, UsdSkel, UsdUI, UsdVol
 
+
+# Custom control, but does not have any UI. Instead we use
+# this control to be notified from UFE when any attribute has changed
+# so we can update the AE. This is to fix refresh issue
+# when transform is added to a prim.
+class UfeAttributesObserver(ufe.Observer):
+    def __init__(self, item):
+        super(UfeAttributesObserver, self).__init__()
+        self._item = item
+
+    def __del__(self):
+        ufe.Attributes.removeObserver(self)
+
+    def __call__(self, notification):
+        if isinstance(notification, ufe.AttributeValueChanged):
+            if notification.name() == "xformOpOrder":
+                mel.eval("evalDeferred(\"AEbuildControls\");")
```

Comment:
using updateAE(node) force the prim tab to become the active one.
AEbuildControls Look to be a better choice to force the template to be rebuild